easybuild/easyblocks/boost.py

```diff
@@ -1,240 +0,0 @@
-##
```

Comment:
Can you please check if this won't break boost builds on other systems?

Reply:
See comment above